usr/libexec/gamescope-sdl-workaround

```diff
@@ -3,20 +3,22 @@
 
 # Regexes for GPUs requiring the SDL backend workaround
 # Can be used in regexes as $POLARIS_RE|$VEGA_RE etc for readability
-# Needed output for a regex can gathered from "switcherooctl list"
-POLARIS_RE="Baffin|Ellesmere|Lexa PRO|Polaris \d{2} XL|RX 460/560D / Pro 450/455/460/555/555X/560/560X|RX 470/480/570/570X/580/580X/590|540/540X/550/550X|RX 540X/550/550X"
+# Needed output for a regex can gathered from "lspci -nnk | grep -P "(VGA compatible|3D) controller" -A1"
+# The regex format is Perl REGEX
+POLARIS_RE="Ellesmere|Lexa (\[|XT|XL|PRO)|Baffin|Polaris \d{2}|Polaris\d{2}|RX 460/560D / Pro 450/455/460/555/555X/560/560X|RX 470/480/570/570X/580/580X/590|540/540X/550/550X|RX 540X/550/550X|RX\s?\d{3}(\s|\])"
```

Comment:
`RX\s?\d{3}(\s|\])`

This makes me a little anxious. Will match any 3 digit number so probably okay.

Reply:
> `RX\s?\d{3}(\s|\])`
> 
> This makes me a little anxious. Will match any 3 digit number so probably okay.

it will match "RX570 whatever" but not "RX5700"
it is based on the PCI ID database list from https://pci-ids.ucw.cz/

Reply:
Change pushed at request of HikariKnight